doc/chapters/debugging/index.rst

```diff
@@ -99,6 +99,27 @@ available as normal without any changes. If you have dynamic libraries inside
 your container that the application are linking to, you may need to add the
 proper `solib-search-paths` to the libraries.
 
+Running GDBserver on application running in container
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you are running `SoftwareContainerAgent` on a target hardware you can still
+debug applications inside the container using `gdbserver` which exports a gdb
+socket on network and then you connect a gdb to that network socket and you can
+debug using. Note that the `gdb` needs to be for the same architecture,
+preferably also built together with the toolchain used to build for the target.
+
+To perform this:
+
+1. Start a container.
+2. Configure network for the container.
+3. Launch the app inside the container using ``gdbserver :7777 <appname>``.
```

Comment:
This assumes one has gdb on the target, right? And that it is in a path that's mounted into the container? All that is fine, just want to make sure.